src/main/java/nl/tudelft/jpacman/level/Level.java

```diff
@@ -353,18 +363,22 @@ public void run() {
 	 * 
 	 * @author Jeroen Roosen 
 	 */
-	public interface LevelObserver {
+	public interface WinObserver<T> {
```

Comment:
I think that `LevelObserver` actually makes more sense for an observer that observes both wins and loses.